tools-local/Microsoft.ML.CodeAnalyzer/Microsoft.ML.CodeAnalyzer.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.8.2" />
```

Comment:
Since all our projects (including our `.nupkgproj` projects) have a reference to this .csproj, we should ensure we are not getting dependencies on these packages in our nuget packages, and that our built `.dll`s don't have references to these assemblies (and that they don't show up in Intellisense at all, so no one makes the mistake of referencing these assemblies).

Reply:
I had the same thought.  I trust MSBuild and VisualStudio will do the right thing with this, but I don't trust NuGet.

Reply:
Hi @eerhardt that's a good point. As near as I can . The `build.cmd -release -buildPackages`. The analyzers are not part of the content or the nuget dependencies. Also if I unzip the nugets, and do something like `[Reflection.Assembly]::LoadFrom("...").GetReferencedAssemblies()` on the usual suspects (`Microsoft.ML.Core.dll` and `Microsoft.ML.Data.dll` and the like) I do not see either the project for the analyzer or StyleCop. So, I *think* everything is working as desired, and these are analyzer-only references as intended?

Reply:
Perfect.  Also, check the `.nuspec` file in the `Microsoft.ML` nuget package doesn't have these dependencies either.  If not, then I think we are good to go.